src/V3Life.cpp

```diff
@@ -399,6 +400,18 @@ class LifeVisitor final : public VNVisitor {
         bodyLifep->lifeToAbove();
         VL_DO_DANGLING(delete bodyLifep, bodyLifep);
     }
+    virtual void visit(AstDelay* nodep) override {
+        // V3Life doesn't understand time sense - don't optimize
+        m_lifep->clear();
+        m_noopt = true;
```

Comment:
Suggest combine these two into a function, as any time we set m_noopt we can clear.